dtmi/com/example/azuresphere/thermometer-1.json

```diff
@@ -0,0 +1,75 @@
+{
+    "@context": [
+        "dtmi:iotcentral:context;2",
```

Comment:
consider removing this line, not needed unless you want to use central specific semantic types